### ISSUE TITLE
[REF-553] Avoid websocket connection if the app does not use State

### ISSUE THREAD

**Issue by include** [reflex-dev/reflex#1771]
**Describe the bug**

I've build a sample website and deployed the frontend only to Cloudflare (uploading the frontend.zip). I am not using any states and I can see the page loads without any visible problem - but - in the console - I see that it is trying to establish a websocket connection. 

**To Reproduce**

- Step 1: Setup the project
```
mkdir demo && cd demo
reflext init
```

- Step 2: Comment the state Class in demo.py
```
# class State(rx.State):
#     pass
```

- Step 3: Generate the frontend zip file
```
reflex export --frontend-only
```

- Step 4: Take the zip file and upload it to Cloudflare (or open it in local browser).

**Expected behavior**
No errors in the developer tools console.

**Specifics (please complete the following information):**
 - Python Version: 3.11.5
 - Reflex Version: 0.2.6
 - OS: macOS Ventura 13.5.1
 - Browser (Optional): Brave 1.57.57
 
**Additional context**
Currently I just need the the simplicity of Reflex to build a static website and deploy it to Cloudflare pages. Can't we use Reflex to build static websites?

Thanks in advance!
F

<sub>[REF-553](https://linear.app/reflex-dev/issue/REF-553/websocket-connection-to)</sub>

**Comment by masenf** [comment]
We were actually discussing this yesterday amongst the team. I think it's something that we should support... essentially if the `App` does not specify a `State`, then it should go into "static mode" and not try to connect to the backend.

Will try to pick it up for a future release.

**Comment by include** [comment]
Hi @masenf - thank you for your feedback. Really appreciate your team is considering this! As far as I can see this will open other use cases for Reflex. Looking forward to this feature :)

**Comment by wassafshahzad** [comment]
@masenf  I can take a look at it..

**Comment by masenf** [comment]
fixed by #1950 and #1978
